pkg/kubelet/volume_host.go

```diff
@@ -80,6 +80,7 @@ func NewInitializedVolumePluginMgr(
 
 // Compile-time check to ensure kubeletVolumeHost implements the VolumeHost interface
 var _ volume.VolumeHost = &kubeletVolumeHost{}
+var _ volume.KubeletVolumeHost = &kubeletVolumeHost{}
```

Comment:
I think with your change, we now expect `runtimeState` in kubelet to set initialized before `NewInitializedVolumePluginMgr` is called.

We should add a comment to emphasize that to avoid panic in the future when people move code around. It is not obvious from the code.

Reply:
I think `runtimeState` in Kubelet for `storageErr` is populated *during* `NewInitializedVolumePluginMgr`. Could you clarify what kind of comment you're looking for and where? What is the unexpected event/state here

Reply:
> We should add a comment to emphasize that to avoid panic in the future when people move code around. It is not obvious from the code.

is there a reason not to populate runtimeState as part of the kubelet struct creation:

```
klet := &Kubelet{
  ...
  runtimeState: newRuntimeState(maxWaitForContainerRuntime),
  ...
}
```

that would ensure SetKubeletError couldn't be called on the kubelet struct before runtimeState was set